charts/opentelemetry-collector/templates/_pod.tpl

```diff
@@ -110,6 +110,25 @@ containers:
       httpGet:
         path: {{ .Values.readinessProbe.httpGet.path }}
         port: {{ .Values.readinessProbe.httpGet.port }}
+    startupProbe:
```

Comment:
To make this an addative feature, lets only set `startupProbe` if `.Values.startupProbe` is set and lets update the values.yaml to not set `startupProbe` by default.

Reply:
Done!

Do you want me to add `startupProbe` values to an existing / new example?

Reply:
No, it is simple enough that we don't need to.